mipidsi/src/lib.rs

```diff
@@ -289,4 +289,12 @@ where
         self.sleeping = false;
         Ok(())
     }
+
+    ///
+    /// Returns the DCS interface for sending raw commands.
+    /// *WARNING*: user is responsible for ensuring display state validity
+    ///
```

Comment:
```suggestion
    /// Returns the DCS interface for sending raw commands.
    ///
    /// # Safety
    ///
    /// Sending raw commands to the controller can lead to undefined behaviour,
    /// because the rest of the code isn't aware of any state changes that were caused by sending raw commands.
    /// The user must ensure that the state of the controller isn't altered in a way that interferes with the normal
    /// operation of this crate.
```
Changed `*WARNING*` into `# Safety`, because of the API guidelines: https://rust-lang.github.io/api-guidelines/documentation.html#function-docs-include-error-panic-and-safety-considerations-c-failure